bin/git_hooks.dart

```diff
@@ -8,19 +8,24 @@ import 'package:git_hooks/install/create_hooks.dart';
 void main(List<String> arguments) {
   if (arguments.isNotEmpty) {
     var str = arguments[0];
-    if (arguments != null && arguments.isNotEmpty) {
+    if (arguments.isNotEmpty) {
       if (str == 'create') {
         //init files
-        var targetPath = arguments[1];
-        if (targetPath != null && targetPath.endsWith('.dart')) {
+        String? targetPath;
+        try {
+          targetPath = arguments[1];
+        } on RangeError {
+          targetPath = null;
+        }
+        if (targetPath is String && targetPath.endsWith('.dart')) {
```

Comment:
This accounts for when there is no `arguments[1]`. This now throws `RangeError` instead of just returning `null`.